### PR TITLE
Add HPCG/3.1 with GCC/10.3.0 to NESSI/2022.11

### PR DESCRIPTION
**Pull request by TopRichard**
Trying to add HPCG/3.1 with GCC/10.3.0 to NESSI/2022.11

### CODE REVIEW

File: EESSI-pilot-install-software.sh [hunk 1]
```diff
@@ -464,6 +464,12 @@ $EB ImageMagick-7.0.11-14-GCCcore-10.3.0.eb --robot
 #cat $($EB --last-log)
 #check_exit_code $exit_code "${ok_msg}" "${fail_msg}"
 
+#echo "Installing HPCG/3.1 for GCC/10.3.0..."
+#ok_msg="HPCG installed. Nice!"
+#fail_msg="Installation of HPCG failed, that's unexpected..."
+$EB  HPCG-3.1-foss-2021a.eb --robot
+#check_exit_code $exit_code "${ok_msg}" "${fail_msg}"
```
Comment (trz42) [inline EESSI-pilot-install-software.sh]: Any reason to leave lines 467-469 & 471 commented out?

**Review by trz42 — COMMENTED** [review]
Maybe the event was not forwarded from the smee server to the bot instances?

File: EESSI-pilot-install-software.sh (hunk 1 above)
Comment (TopRichard) [reply EESSI-pilot-install-software.sh]: left behind by mistake

File: EESSI-pilot-install-software.sh [hunk 2]
```diff
@@ -468,6 +468,12 @@ $EB ImageMagick-7.0.11-14-GCCcore-10.3.0.eb --robot
 #cat $($EB --last-log)
 #check_exit_code $exit_code "${ok_msg}" "${fail_msg}"
 
+#echo "Installing HPCG/3.1 for GCC/10.3.0..."
+ok_msg="HPCG installed, Nice!"
+fail_msg="Installation of HPCG failed, that's unexpected..."
+$EB  HPCG-3.1-foss-2021a.eb --robot
+check_exit_code $exit_code "${ok_msg}" "${fail_msg}"
```
Comment (trz42) [inline EESSI-pilot-install-software.sh]: `exit_code` is not set after the `$EB` command. Something like

```bash
exit_code=$?
```

is missing.

Alternatively, replace `$exit_code` with `$?`.

**Review by trz42 — CHANGES_REQUESTED** [review]

File: EESSI-pilot-install-software.sh (hunk 2 above)
Comment (TopRichard) [reply EESSI-pilot-install-software.sh]: Copied the syntax from Cadical and missed the variable exit_code=$?, will replace and build again

**Pull request closed without merging**